### PR TITLE
Update customer-key-set-up.md

### PR DESCRIPTION
**Pull request by cgill**
Good afternoon, folks! I was working today to set up Customer Key and found that the FastTrack portal has changed A LOT, now showing information cards instead of the traditional offers. I have made changes to address the steps necessary to properly fulfill submitting an offer for Customer Key, and hope you agree that these are the necessary changes. Thanks in advance!

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/customer-key-set-up.md [hunk 1]
```diff
@@ -103,15 +103,15 @@ To submit an offer to activate Customer Key, complete these steps:
...
 2. Once you're logged in, browse to the **Dashboard**.
 
-3. Choose **Offers**, and review the list of current offers.
+3. Choose **Deploy** from the navigation bar **OR** select **View all deployment resources** on the **Deploy** information card, and review the list of current offers.
 
-4. Choose **Learn More** for the offer that applies to you:
+4. Choose the information card for the offer that applies to you:
 
-   - **Exchange Online and Skype for Business:** Choose **Learn More** on the **Customer Key for Exchange** offer.
+   - **Exchange Online and Skype for Business:** Choose **Request encryption key help for Exchange online** offer.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
   - **Exchange Online and Skype for Business:** Choose the **Request encryption key help for Exchange online** offer.
```

File: microsoft-365/compliance/customer-key-set-up.md [hunk 2]
```diff
@@ -103,15 +103,15 @@ To submit an offer to activate Customer Key, complete these steps:
...
 
-   - **SharePoint Online, OneDrive for Business, and Teams files:** Choose **Learn More** on the **Customer Key for SharePoint and OneDrive for Business** offer.
+   - **SharePoint Online, OneDrive for Business, and Teams files:** Choose **Request encryption key help for Sharepoint and OneDrive** offer.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
   - **SharePoint Online, OneDrive, and Teams files:** Choose the **Request encryption key help for Sharepoint and OneDrive** offer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@cgill  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review.  


Thank you.

cc: @kenwith

**Pull request merged**